python/datasources/acs_population.py

```diff
@@ -0,0 +1,376 @@
+        stored. """
+        # Writes multiple tables, so this is not applicable.
+        pass
+
+    def upload_to_gcs(self, url, gcs_bucket, filename):
+        """Uploads population data from census to GCS bucket."""
+        metadata = fetch_acs_metadata(self.base_acs_url)
+        var_map = parse_acs_metadata(metadata, list(GROUPS.keys()))
+
+        concepts = list(SEX_BY_AGE_CONCEPTS_TO_RACE.keys())
+        concepts.append(HISPANIC_BY_RACE_CONCEPT)
+
+        for concept in concepts:
+            group_vars = get_vars_for_group(concept, var_map, 2)
+            cols = list(group_vars.keys())
+            url_params = get_census_params(cols, self.county_level)
+            url_file_to_gcs.url_file_to_gcs(
+                self.base_acs_url, url_params, gcs_bucket, get_filename(concept))
+
+        url_params = get_census_params([TOTAL_POP_VARIABLE_ID], self.county_level)
+        url_file_to_gcs.url_file_to_gcs(
+            self.base_acs_url, url_params, gcs_bucket, TOTAL_POP_FILE_NAME)
+
+    def write_to_bq(self, dataset, gcs_bucket, filename):
+        """Writes population data to BigQuery from the provided GCS bucket
+
+        dataset: The BigQuery dataset to write to
+        table_name: The name of the biquery table to write to
+        gcs_bucket: The name of the gcs bucket to read the data from
+        filename: The name of the file in the gcs bucket to read from"""
+        # TODO change this to have it read metadata from GCS bucket
+        metadata = fetch_acs_metadata(self.base_acs_url)
+        var_map = parse_acs_metadata(metadata, list(GROUPS.keys()))
+
+        race_and_hispanic_frame = gcs_to_bq_util.load_values_as_dataframe(
+            gcs_bucket, get_filename(HISPANIC_BY_RACE_CONCEPT))
+        race_and_hispanic_frame = update_col_types(race_and_hispanic_frame)
+
+        race_and_hispanic_frame = standardize_frame(
+            race_and_hispanic_frame,
+            get_vars_for_group(HISPANIC_BY_RACE_CONCEPT, var_map, 2),
+            [HISPANIC_COL, RACE_COL],
+            self.county_level,
+            POPULATION_COL)
+
+        total_frame = gcs_to_bq_util.load_values_as_dataframe(
+            gcs_bucket, TOTAL_POP_FILE_NAME)
+        total_frame = update_col_types(total_frame)
+        total_frame = standardize_frame(
+            total_frame,
+            {TOTAL_POP_VARIABLE_ID: ['Total']},
+            [RACE_OR_HISPANIC_COL],
+            self.county_level,
+            POPULATION_COL)
+
+        sex_by_age_frames = {}
+        for concept in SEX_BY_AGE_CONCEPTS_TO_RACE:
+            sex_by_age_frame = gcs_to_bq_util.load_values_as_dataframe(
+                gcs_bucket, get_filename(concept))
+            sex_by_age_frame = update_col_types(sex_by_age_frame)
+            sex_by_age_frames[concept] = sex_by_age_frame
+
+        geo_suffix = "_county" if self.county_level else "_state"
+        frames = {
+            BASE_TABLE_NAME_BY_RACE + geo_suffix: self.get_all_races_frame(
```

Comment:
The exporter is expecting the _std to be the suffix for standardized tables to be exported to csv.

Reply:
Oops I forgot I put the geo suffix after "_std". Fixed